coderd/workspacebuilds_test.go

```diff
@@ -1334,29 +1342,63 @@ func TestWorkspaceBuildTimings(t *testing.T) {
 			ResourceID: resource.ID,
 		})
 
-		// Then: return a response with empty agent script timings
+		// When: fetching timings for the build
 		ctx, cancel := context.WithTimeout(context.Background(), testutil.WaitLong)
 		t.Cleanup(cancel)
 		res, err := client.WorkspaceBuildTimings(ctx, build.ID)
 		require.NoError(t, err)
+
+		// Then: return a response with empty agent script timings
 		require.Empty(t, res.AgentScriptTimings)
 	})
 
 	// Some workspaces might not have agents. It is improbable, but possible.
 	//nolint:paralleltest
 	t.Run("NoAgents", func(t *testing.T) {
-		// When: fetching timings for a build with no agents
+		// Given: a build with no agents
 		build := makeBuild()
 		dbgen.WorkspaceResource(t, db, database.WorkspaceResource{
 			JobID: build.JobID,
 		})
 
-		// Then: return a response with empty agent script timings
-		// trigger build
+		// When: fetching timings for the build
 		ctx, cancel := context.WithTimeout(context.Background(), testutil.WaitLong)
 		t.Cleanup(cancel)
 		res, err := client.WorkspaceBuildTimings(ctx, build.ID)
 		require.NoError(t, err)
+
+		// Then: return a response with empty agent script timings
 		require.Empty(t, res.AgentScriptTimings)
 	})
+
+	//nolint:paralleltest
```

Comment:
Howcome these are not in parallel?

All mentions of `nolint` should be followed with a reason like:

`//nolint:paralleltest // Reason.`

Reply:
I'm just following what we are already doing in other places:
- https://github.com/coder/coder/blob/3e1170c1730cea1d3cf01d1c8ca854584b2f0f4d/coderd/workspacebuilds_test.go#L1186
- https://github.com/coder/coder/blob/3e1170c1730cea1d3cf01d1c8ca854584b2f0f4d/coderd/workspacebuilds_test.go#L1240
- https://github.com/coder/coder/blob/3e1170c1730cea1d3cf01d1c8ca854584b2f0f4d/coderd/workspacebuilds_test.go#L1255

All of them have the same reason based on [this PR ](https://github.com/coder/coder/pull/15069) even thinking I had fixed it [here](https://github.com/coder/coder/pull/15077).